installer/Product.wxs

```diff
@@ -0,0 +1,561 @@
+			</Component>
+			<Component Id="EXPORT_PROGRAM_INFO.EXE" DiskId="1" Guid="{7AF9A30B-D7BF-4FD2-88C0-C2947ED55DCD}">
+				<File Id="EXPORT_PROGRAM_INFO.EXE" Name="export_program_info.exe" Source="$(var.export_program_info.TargetDir)export_program_info.exe" />
+			</Component>
+			<Component Id="EXPORT_PROGRAM_INFO.PDB" DiskId="1" Guid="{88356FE4-6247-4876-BA14-42EDA88CC5C7}">
+				<File Id="EXPORT_PROGRAM_INFO.PDB" Name="export_program_info.pdb" Source="$(var.export_program_info.TargetDir)export_program_info.pdb" />
+			</Component>
+		</ComponentGroup>
+
+		<!--Clear/Setup eBPF store-->
+		<SetProperty Id="Clear_eBPF_store" Value='"[#EXPORT_PROGRAM_INFO.EXE]" --clear' Before="Clear_eBPF_store" Sequence="execute"/>
+		<CustomAction Id="Clear_eBPF_store" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="deferred" Return="check" Impersonate="no"/>
+		<SetProperty Id="Clear_eBPF_store_uninstall" Value='"[#EXPORT_PROGRAM_INFO.EXE]" --clear' Before="Clear_eBPF_store" Sequence="execute"/>
+		<CustomAction Id="Clear_eBPF_store_uninstall" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="deferred" Return="ignore" Impersonate="no"/>
+		<SetProperty Id="Setup_eBPF_store" Value='"[#EXPORT_PROGRAM_INFO.EXE]"' Before="Setup_eBPF_store" Sequence="execute"/>
+		<CustomAction Id="Setup_eBPF_store" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="deferred" Return="check" Impersonate="no"/>
+
+		<!--Install/Uninstall netsh helper-->
+		<!--qtexec does not currently support a working directory (ref. https://github.com/wixtoolset/issues/issues/1265)-->
+		<CustomAction Id="eBPF_netsh_helper_install" ExeCommand='netsh add helper ebpfnetsh.dll' Directory="INSTALLFOLDER" Execute="deferred" Return="check" Impersonate="no"/>
+		<CustomAction Id="eBPF_netsh_helper_uninstall" ExeCommand='netsh delete helper ebpfnetsh.dll' Directory="INSTALLFOLDER" Execute="deferred" Return="ignore" Impersonate="no"/>
+		<CustomAction Id="eBPF_netsh_helper_uninstall_rollback" ExeCommand='netsh delete helper ebpfnetsh.dll' Directory="INSTALLFOLDER" Execute="rollback" Return="ignore" Impersonate="no"/>
+
+		<!--Install/Uninstall eBPF Service -->
+		<?if $(var.Configuration) = Debug ?>
+		<ComponentGroup Id="eBPF_Service" Directory="dir_JIT">
+			<Component Id="EBPFSVC.PDB" DiskId="1" Guid="{D1935DF0-2FC7-42F5-81E5-19AF88D6244B}">
+				<File Id="EBPFSVC.PDB" Name="EbpfSvc.pdb" Source="$(var.ebpfsvc.TargetDir)ebpfsvc.pdb" />
+			</Component>
+			<Component Id="eBPF_Service" Guid="{25F4F9A3-78EB-4598-9982-98FB65377A51}">
+				<File Source="$(var.ebpfsvc.TargetPath)" KeyPath="yes" />
+				<ServiceInstall Id="eBPFSvcInstaller" Name="eBPFSvc" DisplayName="eBPF Service" Description="eBPF Service"
+								Start="auto"
+								Type="ownProcess"
+								ErrorControl="normal"
+								Account="[WIX_ACCOUNT_LOCALSERVICE]"
+								Vital="yes"
+								Interactive="no" />
+				<ServiceControl Id="eBPFSvc_Start" Name="eBPFSvc"
+								Start="install"
```

Comment:
I am not sure I understand. The command `eBPFSvc -Install` installs the service, not start it.

Reply:
The service is currently set to install as auto-start, i.e. `Start="auto"` within the `Serviceinstall` tag.

Setting `Start='install'`, `Stop="both"` and `Remove="uninstall"` within the `ServiceControl` tag would mean: "*start the service on install, remove the service when the product is uninstalled, and stop the service both on install and uninstall*".
`ServiceControl` is not to be confused with an `sc` ovveride.